Patas.Em.Harmonia.Domain/Models/Entities/NgoData.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Patas.Em.Harmonia.Domain.Models.Entities
+{
+    public class NgoData
```

Comment:
esse carinha é o que vem do controller? se for, acha melhor por como DTO não? tipo são nomenclaturas, como tem várias, tipo ViewModel e etc, mas se ele vier do controler a melhor nomenclatura e pasta que ele deve ficar é no DTO

Reply:
Resolvido!